core/src/test/java/io/questdb/test/cairo/mv/MatViewTest.java

```diff
@@ -66,11 +76,22 @@ public static void setUpStatic() throws Exception {
         AbstractCairoTest.setUpStatic();
     }
 
+    @Parameterized.Parameters(name = "rows_per_query={0}")
```

Comment:
We don't need all 50 tests in both modes, make this parameter random.

Reply:
Addressed in 8d17c21d7ccd2f9784042da87b26aeed39f4fe4f